pkg/builtins/bitwise.go

```diff
@@ -0,0 +1,104 @@
+package builtins
+
+import (
+	"github.com/lambdaclass/cairo-vm.go/pkg/lambdaworks"
+	"github.com/lambdaclass/cairo-vm.go/pkg/vm/memory"
+	"github.com/pkg/errors"
+)
+
+const BITWISE_BUILTIN_NAME = "bitwise"
+const BITWISE_CELLS_PER_INSTANCE = 5
+const BITWISE_TOTAL_N_BITS = 251
+const BIWISE_INPUT_CELLS_PER_INSTANCE = 2
+
+type BitwiseBuiltinRunner struct {
+	base     memory.Relocatable
+	included bool
+}
+
+func BitwiseError(err error) error {
+	return errors.Wrapf(err, "Bitwise builtin error\n")
+}
+
+func ErrFeltBiggerThanPowerOfTwo(felt lambdaworks.Felt) error {
+	return BitwiseError(errors.Errorf("Expected felt %d to be smaller than  2**%d", felt, BITWISE_TOTAL_N_BITS))
+}
+
+func NewBitwiseBuiltinRunner(included bool) *BitwiseBuiltinRunner {
+	return &BitwiseBuiltinRunner{
+		included: included,
+	}
+}
+
+func (b *BitwiseBuiltinRunner) Base() memory.Relocatable {
+	return b.base
+}
+
+func (b *BitwiseBuiltinRunner) Name() string {
+	return BITWISE_BUILTIN_NAME
+}
+
+func (b *BitwiseBuiltinRunner) InitializeSegments(segments *memory.MemorySegmentManager) {
+	b.base = segments.AddSegment()
+}
+
+func (b *BitwiseBuiltinRunner) InitialStack() []memory.MaybeRelocatable {
+	if b.included {
+		return []memory.MaybeRelocatable{*memory.NewMaybeRelocatableRelocatable(b.base)}
+	} else {
+		return []memory.MaybeRelocatable{}
+	}
+}
+
+func (b *BitwiseBuiltinRunner) DeduceMemoryCell(address memory.Relocatable, segments *memory.Memory) (*memory.MaybeRelocatable, error) {
```

Comment:
```suggestion
func (b *BitwiseBuiltinRunner) DeduceMemoryCell(address memory.Relocatable, memory *memory.Memory) (*memory.MaybeRelocatable, error) {
```
Wrong name, `mem` could also be used if it conflicts with the package name